README.md

```diff
@@ -587,6 +587,8 @@ spec:
       # false (default) = Docker support is provided by a sidecar container deployed in the runner pod.
       # true = No docker sidecar container is deployed in the runner pod but docker can be used within teh runner container instead. The image summerwind/actions-runner-dind is used by default.
       dockerdWithinRunnerContainer: true
+      # Optional Docker registry mirror, only applicable if dockerdWithinRunnerContainer = true
+      dockerRegistryMirror: https://mirror.gcr.io/
```

Comment:
@rofafor Worth adding mention to https://cloud.google.com/container-registry/docs/pulling-cached-images for more information? I'd appreciate it if you could add it here or to our docs in another PR